.eslintrc.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+    env: {
+      es2021: true,
+      node: true
+    },
+    extends: 'airbnb-typescript',
+    overrides: [
+      {
+        env: {
+          node: true
+        },
+        files: [
+          '.eslintrc.{js,cjs}'
+        ],
+        parserOptions: {
+          sourceType: 'module'
+        }
+      }
+    ],
+    parserOptions: {
+      ecmaVersion: 'latest',
+      sourceType: 'module'
+    },
+    rules: {
+      'comma-style': ['error', 'last'], // requires a comma after and on the same line as an array element, object property, or variable declaration
+      'comma-dangle': ['error', 'never'], // disallow trailing commas
+      'no-use-before-define': [
+          // warns when it encounters a reference to an identifier that has not yet been declared.
+          'error',
+          {
+            functions: true,
+            classes: true,
+            variables: false
+          }
+        ],
+    }
+  };
+  
```

Comment:
```suggestion
  };

```
There seems to be some extra space. Kindly take them out.